src/rust/lib/debug-scenes/src/shapes.rs

```diff
@@ -26,6 +28,20 @@ pub fn run_example_shapes() {
     mem::forget(app);
 }
 
+fn set_up_visualization_demo_frp(graph_editor:&GraphEditor) {
+    let network       = &graph_editor.frp.network;
```

Comment:
I know it doesnt matter too much here, but this is almost always a bad idea to extend a network not owned by you. Create a new network here instead and own it in a variable instead please .